src/components/Book.jsx

```diff
@@ -1,29 +1,91 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
 import { deletedBook } from '../redux/books/bookSlice';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
+  const [percent, setPercent] = useState(0);
 
   const deleteBook = () => {
     dispatch(deletedBook(book.item_id));
   };
 
+  const numbers = [];
+
+  const storeNumbers = () => {
+    let i = 0;
+    // eslint-disable-next-line no-plusplus
+    for (i; i <= 100; i++) {
+      numbers.push(i);
+    }
+    return numbers;
+  };
+
+  storeNumbers();
+
+  useEffect(() => {
+    const random = numbers[Math.floor(Math.random() * numbers.length)];
+    setPercent(random);
+  }, []);
+
   return (
-    <div className="book-card border bg-white p-3 shadow rounded">
-      <small>{book.category}</small>
-      <h2 className="title text-blue-900 font-extrabold text-xl">
-        {book.title}
-      </h2>
-      <p className="author">{book.author}</p>
-      <button
-        type="button"
-        className="p-2 text-white mt-3 bg-blue-600 rounded"
-        onClick={deleteBook}
-      >
-        Remove
-      </button>
+    <div className="book-card border bg-white p-8 shadow-sm rounded flex items-center justify-between">
+      <div>
+        <p className="text-gray-400 font-bold">{book.category}</p>
+        <h1 className="title text-slate-700 font-extrabold text-2xl">
+          {book.title}
+        </h1>
+        <p className="author text-blue-700 font-light text-lg">{book.author}</p>
+        <div className="actions mt-5">
+          <button
+            type="button"
+            className="pr-2 text-blue-700 text-sm hover:text-blue-900"
+          >
+            Comments
+          </button>
+          <button
+            type="button"
+            className="px-2 text-blue-700 border-l-2 border-r-2 text-sm hover:text-blue-900"
+            onClick={deleteBook}
+          >
+            Remove
+          </button>
+          <button
+            type="button"
+            className="px-2 text-blue-700 text-sm hover:text-blue-900"
+          >
+            Edit
+          </button>
```

Comment:
- [ ] REQUIRED] The button (comments, remove, edit) also needs to follow the exact styles as used in the zeplin design. Kindly change the font, color, font weight, and every property they have on zeplin.
- [ ] [REQUIRED] Kindly apply the same to the other h-tags (Fiction, dfsdf, sdfsdfsd, chapter 3, current chapter), update-progress-button.

### Your design
<img width="1440" alt="Screenshot 2022-11-08 at 18 45 20" src="https://user-images.githubusercontent.com/58692590/200642496-e1b6777c-13b1-4f32-aeb8-c0b7ceb97d55.png">

### Zeplin Design
<img width="1020" alt="Screenshot 2022-11-08 at 19 14 21" src="https://user-images.githubusercontent.com/58692590/200643509-1a4de402-b06b-4d6c-9265-40ee2559b262.png">